### DISCUSSION TITLE
A faster website! (aka defer.js)

### DISCUSSION THREAD

**Discussion by beinhd** [shinsenter/defer.js#124] [Q&A]
incredibly practical and efficient
 but the issue that i face is the checkout page not upload and paypal not appear
![Screenshot 2023-09-16 181631](https://github.com/shinsenter/defer.js/assets/30273926/224659f9-3b7b-4b39-9571-9b046e8af68b)

**Comment by shinsenter** [comment]
@beinhd 

Hello, sorry for the late response.

I need more information about how you are using this library for your website, for example which version of the plugin you are using, and if there are any errors showing up in the Console tab of the Web Inspector feature of your browser?

Best regards
